core/domain/stats_jobs_one_off.py

```diff
@@ -679,47 +673,24 @@ def entity_classes_to_map_over(cls):
 
     @classmethod
     def require_non_negative(
-            cls, exp_id, exp_version, property_name, value, state_name=None):
+            cls, exp_id, exp_version, property_name, value,
```

Comment:
This is the change we talked about offline. `require_non_negative` used as a generator that is why this function was not getting called